rc-events-service/src/main/scala/com/epam/random_coffee/events/services/EventService.scala

```diff
@@ -0,0 +1,18 @@
+package com.epam.random_coffee.events.services
+
+import com.epam.random_coffee.events.api.request.{ CreateEventRequest, UpdateEventRequest }
+import com.epam.random_coffee.events.api.response.EventResponse
+import com.epam.random_coffee.events.model.Event
+
+import scala.concurrent.Future
+
+trait EventService {
+
+  def create(event: CreateEventRequest): Future[EventResponse]
+
+  def find(id: Int): Future[Option[Event]]
```

Comment:
Here we should return Future[Option[EventResponse]]

Reply:
@myazinn We discussed last week about mapping to Response at service's level instead do it straight in controllers, so maybe it would be better to add one additional service which would just map results from lower services to responses and then we would use it in controllers.

Reply:
@OlegKoshelev let's discuss it tomorrow

Reply:
@myazinn Sure)

Reply:
As we discussed, find method return Option[Event], Create - event, update - event, delete - unit.